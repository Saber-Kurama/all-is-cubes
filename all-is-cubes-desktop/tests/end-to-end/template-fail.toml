bin.name = "all-is-cubes"
args = "--no-config-files --simplify-log-format --template fail --graphics headless"
status.code = 1
