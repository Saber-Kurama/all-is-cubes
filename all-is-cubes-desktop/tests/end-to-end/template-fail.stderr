[DEBUG] (1) Initialized session [..]
[INFO] Randomly chosen universe seed: [..]
Error: failed to create universe from requested template or file

Caused by:
    0: failed while constructing universe from template Fail
    1: An error occurred while generating object 'space'
    2: the Fail template always fails to generate
